test/linalg/svd.jl

```diff
@@ -4,6 +4,31 @@ using Base.Test
 
 using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
+@testset "Simple svdvals / svdfact tests" begin
+    ≈(x,y) = isapprox(x,y,rtol=1e-15)
```

Comment:
I just started using `\approxeq` in `StaticArrays` for exactly similar purposes.  I'll go with that I suppose, unless the `\equivDD` symbol is already conventional for the "almost eps" comparison?